docker/docker-compose.bridged.yml

```diff
@@ -15,6 +17,7 @@ services:
       - main
       - Node1
     environment:
+      - PUBLIC_VALIDATOR_ADDRESS=Node1:30344
```

Comment:
nit: does port number need to increase for every node since it is prefixed anyway with Node{Id}?

Reply:
It does not, but all nodes listen on different ports, so we would have to override that too. I only intended to fix deployment, not organize our docker-compose.